README.md

```diff
@@ -25,23 +25,19 @@ Built with the [Meltano SDK](https://sdk.meltano.com) for Singer Taps and Target
 | database                     | False    | None    | Database name. Note if sqlalchemy_url is set this will be ignored. |
 | sqlalchemy_url               | False    | None    | SQLAlchemy connection string. This will override using host, user, password, port, dialect, and all ssl settings. Note that you must escape password special characters properly. See https://docs.sqlalchemy.org/en/20/core/engines.html#escaping-special-characters-such-as-signs-in-passwords |
 | dialect+driver               | False    | postgresql+psycopg2 | Dialect+driver see https://docs.sqlalchemy.org/en/20/core/engines.html. Generally just leave this alone. Note if sqlalchemy_url is set this will be ignored. |
-| default_target_schema        | False    | None    | Postgres schema to send data to, example: tap-clickup |
-| hard_delete                  | False    |       0 | When activate version is sent from a tap this specefies if we should delete the records that don't match, or mark them with a date in the `_sdc_deleted_at` column. |
+| default_target_schema        | False    | melty   | Postgres schema to send data to, example: tap-clickup |
+| activate_version             | False    |       1 | If set to false, the tap will ignore activate version messages. If set to true, add_record_metadata must be set to true as well. |
```

Comment:
```suggestion
| activate_version             | False    |       True  | If set to false, the tap will ignore activate version messages. If set to true, add_record_metadata must be set to true as well. |
```

SDK bug for generating Readmes for booleans, we should consistently use True/False I think

Reply:
Fixed.